### ISSUE TITLE
APIs used / not used

### ISSUE THREAD

**Issue by distributev** [distributev/the-app#86]
Which of the following APIs are actually used in GUI code and which are not used (is there any API which is not used in the code but was added only for the "API completeness" reason)?

Below I see many APIs which looks like they are not used at all (please don't change the code - only list used / not used APIs and I will think to an approach)

GET /api/flows/ => Get a list of all flows
GET /api/flows/:id => Get flow :id details
POST /api/flows/ => Create a new flow
PUT /api/users/:id => Update flow :id
PATCH /api/users/:id => Update flow :id
DELETE /api/flows/:id => Delete flow with :id

GET /api/flows/mail-merge => Get a list of mail-merge flows

POST /api/flows/mail-merge => Create a new mail-merge flow
GET /api/flows/mail-merge/:code/history => Get a list of mail-merge flow history by recipient :code
GET /api/flows/mail-merge/:template/config => Get a mail-merge :template config
GET /api/flows/mail-merge/:template/form => Get a mail-merge :template form
GET /api/flows/mail-merge/:template/email => Get a mail-merge :template email html
GET /api/flows/mail-merge/:template/document => Get a mail-merge :template document html

**Comment by luismanuel001** [comment]
Correct, this where created for API completeness, auto generated by the angular-fullstack template generator, I didnt remove them since they might be used later:

```
GET /api/flows/ => Get a list of all flows
GET /api/flows/:id => Get flow :id details
POST /api/flows/ => Create a new flow
PUT /api/flows/:id => Update flow :id
DELETE /api/flows/:id => Delete flow with :id
```

The rest are all used on the GUI.

**Comment by distributev** [comment]
Please put a tag / comment **_NOT USED (auto generated by the angular-fullstack)_** on the APIs which are not used and which are there only for "API completeness"

These will be remove later if not used at all.
